Claudinee11/Property-Pro-Lite1
# deveeloper page

#### What does this PR do?
- change index page
- adds  sold  button 
-add logout
-add multiple images
-add logout

#### Description of Task to be completed
- A user should be able to signup using a form and on clicking the sign up  button, they are redirected to login page.
-user should be able to logout
-user should be able to see more details about the properties
-user should be able to add multiple images

#### How should this be manually tested?
- clone the repo and switch to the branch ft-deveeloper
- start the server
- you should be successfully redirected to home.html




#### Screenshots (if appropriate)
Image of signup page
[Capture](https://user-images.githubusercontent.com/50420731/60887112-6db87c80-a2b8-11e9-80bf-54a886694c2e.PNG)
